AdGuardHomeDisallowedIPs.txt

```diff
@@ -6441,44 +6443,34 @@
 2001:dd9::/48
 2001:df0:27e::/48
 2001:df0:423::/48
+2001:df0:9c0::/48
```

Comment:
Can you provide some verification?

Reply:
This is fetched with my self-created script from pexcn daily list, of which the address can be found in description of this section.